k8s/models/apiextensions_v1_custom_resource_definition.py

```diff
@@ -122,6 +147,10 @@ class CustomResourceValidation(Model):
     openAPIV3Schema = Field(JSONSchemaProps)
 
 
+class CustomResourceValidationStatusEnabled(Model):
+    openAPIV3Schema = Field(JSONSchemaPropsStatusEnabled)
```

Comment:
Since the difference in allowed fields when the status subresource is enabled is only in JSONSchemaProps, maybe `CustomResourceValidationStatusEnabled` can be made unnecessary by letting `CustomResourceValidation.openAPIV3Schema` be either JSONSchemaProps type
```suggestion
    openAPIV3Schema = Field(JSONSchemaProps, alt_type=JSONSchemaPropsStatusEnabled)
```